packages/components/src/internal/components/samples/actions.ts

```diff
@@ -926,3 +926,28 @@ export async function getSelectedSampleTypes(model: QueryModel): Promise<Array<s
         }
     })
 }
+
+export async function getSampleIdsFromSelection(
```

Comment:
This name is very much like the `getSampleRowIdsFromSelection` method, but isn't actually specific to samples in its functionality.  Perhaps rename it to `getFieldRowIdsFromSelection` or `getUniqueFieldRowIdsFromSelection` and rename `sampleFieldKey` to just `fieldKey`?  At least some documentation here would be appreciated.